certbot-apache/certbot_apache/configurator.py

```diff
@@ -511,7 +511,11 @@ def _create_vhost(self, path):
 
         """
         addrs = set()
-        args = self.aug.match(path + "/arg")
+        try:
+            args = self.aug.match(path + "/arg")
+        except RuntimeError:
+            logger.warn("Encountered a problem while parsing file: %s, skipping", path)
```

Comment:
All `logger.warning` messages are shown to the user so it'd be nice if we could show them the unescaped path here. Is there an easy way to do this?